ui/tests/acceptance/mfa-login-test.js

```diff
@@ -18,11 +18,17 @@ module('Acceptance | mfa-login', function (hooks) {
     ENV['ember-cli-mirage'].handler = 'mfaLogin';
   });
   hooks.beforeEach(function () {
+    this.auth = this.owner.lookup('service:auth');
     this.select = async (select = 0, option = 1) => {
       const selector = `[data-test-mfa-select="${select}"]`;
       const value = this.element.querySelector(`${selector} option:nth-child(${option + 1})`).value;
       await fillIn(`${selector} select`, value);
     };
+    return visit('/vault/logout');
+  });
+  hooks.afterEach(function () {
+    // Manually clear token after each so that future tests don't get into a weird state
+    this.auth.deleteCurrentToken();
```

Comment:
This was a weird one. If I tried to visit `logout` in the afterEach hook, one of these tests would fail because it queried the MFA verification more than once. Rather than open that can of worms this fixes the test for now, but we should double-verify with smoke tests that the MFA flow is working as intended. In any case, even if the flow is making multiple requests that shouldn't break the flow